testing/e2e/test-module.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'babel-regenerator-runtime';
```

Comment:
Do we use module-style imports anywhere else in the code?

Reply:
I haven't seen any. I set globals this way instead of using `mochaOptions`'s global property because I need `@babel/register` to load first, in order to use `import`.

Reply:
What I mean is that everywhere else I've seen us using `const someModule = require('some-module');`

Reply:
Gotcha. I retract my statement, we actually use `import`  in /src and our tests files but `require` in /build-system. Looks like I could change it to `require` if I move this file to build-system/tasks/e2e.